### ISSUE TITLE
Wind Rose

### ISSUE THREAD

**Issue by desireeabboud** [ladybug-tools/butterfly#336]
Hello! Thank you for helping us by reporting bugs or requesting for improvements. Before posting your issue, please quickly review the notes below to ensure opening an issue is the right choice for your case.

- If this issue is **not** a bug report or improvement request, please check the [Ladybug Tools forum](http://discourse.ladybug.tools), and start a topic there to discuss your issue. If you don't have an account on our forum, you can ask for one as noted in [this post](http://discourse.ladybug.tools/t/welcome-to-ladybug-tools-forum/8).

- If this issue is related to the Ladybug + Honeybee legacy plugins for Grasshopper, please post it to the repositories for legacy versions: [Ladybug](https://github.com/mostaphaRoudsari/ladybug), [honeybee](https://github.com/mostaphaRoudsari/honeybee). If you're not sure whether you are using the legacy Ladybug + Honeybee, see [here](http://www.ladybug.tools/about.html#14).

- If this issue is for any of the new Plus plugins, please ensure this is the correct repository for your issue. Issues that are specific to the plugin interface should be posted to the correct plugin repository. For example issues related to the Honeybee plugin for Dynamo should be posted to [honeybee-dynamo](https://github.com/ladybug-tools/honeybee-dynamo/) repository and not [honeybee](https://github.com/ladybug-tools/honeybee/).

- Finally, if you haven't done it already, please do a quick search of our existing issues and ensure that the issue has not already been posted.

If you passed all the checks above, feel free to post your issue and please respect these guidelines:
- Be clear about the error and include related information about how to re-create the issue.
- If possible, share an example file or example code for us to recreate the issue.

Thank you again for opening an issue. This will benefit all our users! ALSO DO NOT FORGET TO REMOVE ALL THESE NOTES BEFORE POSTING YOUR ISSUE.

**Comment by desireeabboud** [comment]
hello, why isnt the windrose running/ not displaying on rhino? there are no errors but it does not show.
![ladybug](https://user-images.githubusercontent.com/37541419/37596011-0f42cb48-2b7b-11e8-8630-9ab8789b27c2.png)

**Comment by devngc** [comment]
Hi @desireeabboud,
I am not sure if I see the whole post from you. 
![windrose](https://user-images.githubusercontent.com/20086924/37637382-69684f3c-2c2d-11e8-8768-d4a968fba7c3.png)
But, if this was the rest of the post, then you only need to right-click on the component and click preview. It will then show in Rhino scene.

**Comment by desireeabboud** [comment]
Hello,
It was and I deleted it after I discovered it. I didnt want to bother with a lame question. I just started learning. However, thank you for your help.


Desiree Abboud
LEED Green Associate
Msc. In Sustainable Architecture and Landscape Design
Politecnico di Milano
B.Arch in Architecture and Design
Lebanese American University
Email: desiree.abboud@hotmail.com<mailto:desiree.abboud@hotmail.com>
Tel: +39 3807752063


On 20 Mar,2018, at 6:29 AM, Devang Chauhan <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @desireeabboud<https://github.com/desireeabboud>,
I am not sure if I see the whole post from you.
[windrose]<https://user-images.githubusercontent.com/20086924/37637382-69684f3c-2c2d-11e8-8768-d4a968fba7c3.png>
But, if this was the rest of the post, then you only need to right-click on the component and click preview. It will then show in Rhino scene.
